tests/integration_tests/api/system_backend/test_wrapping.py

```diff
@@ -8,14 +8,12 @@ class TestWrapping(HvacIntegrationTestCase, TestCase):
     TEST_AUTH_METHOD_TYPE = "approle"
     TEST_AUTH_METHOD_PATH = "test-approle"
 
-    def setUp(self):
-        super().setUp()
```

Comment:
What is the reason to remove the `setUp` method?

Reply:
I moved the code from there to the unwrap function because it caused an invalid request error
after i added the test_wrap function. i think this is because it gets called twice. and since the test_wrap
doesn't need it I moved it into the test_unwrap function.